src/main/resources/db/migration/V1_52__cohort_complexity_levels.sql

```diff
@@ -0,0 +1,7 @@
+create table referral_complexity_level_ids (
+    referral_id uuid not null,
+    complexity_level_ids uuid,
```

Comment:
should it not be `complexity_level_id` as there would only ever be one selected?

Reply:
this is the default jpa mapping that means we don't have to add any additional annotations on the entity class. i didn't think too hard about the column naming here.

Reply:
Ah ok. I can foresee that it might be confusing if anyone ever needs to access the database (more with the ids_key than having _ids).

Not sure if support would have readonly access to database to perform any queries?